src/core/Toast/Toaster.tsx

```diff
@@ -18,6 +40,26 @@ export type ToastOptions = Omit<
 export default class Toaster {
   private toasts: ToastProps[] = [];
   private lastId = 0;
+  private settings: ToasterSettings = {
+    order: 'ascending',
+    placement: 'top',
+  };
+
+  /**
+   * Set Toaster settings for toasts order and placement.
+   * Order: ascending or descending.
+   * Placement: 'top' | 'top-start' | 'top-end' | 'bottom' | 'bottom-start' | 'bottom-end';
+   * @param settings
```

Comment:
No need to specify values, as intellisense will already help there.

Also @param is not needed.

Maybe add a line 

```suggestion
   * Set global Toaster settings for toasts order and placement.
   * Settings will be applied to all existing and new toasts on the page.
```

Reply:
Actually my suggestion is wrong, I checked and it does not apply to "existing" toasts, only new ones.

Reply:
Updated description to be only for new toasts